### PR TITLE
Using gradle.properties for tests

### PR DESCRIPTION
**Pull request by rboergertenable**
Changes to move the test config into a gradle.properties file.

### CODE REVIEW

File: src/main/java/com/tenable/io/api/scans/ScansApi.java [hunk 1]
```diff
@@ -330,12 +330,14 @@ public ScanHostDetails hostDetails( int scanId, int hostId ) throws TenableIoExc
      * @param file     The name of the file to import as provided by the response from file: upload
      * @param folderId The id of the destination folder. If not specified, the default folder will be used
      * @param password The password for the file to import (required for nessus.db)
+     * @param aggregate Whether or not the scan details should be included in the Vulnerabilities workbench. (1 if yes, 0 if no)
```
Comment (tenableAndrew) [inline src/main/java/com/tenable/io/api/scans/ScansApi.java]: This was causing import scan results no to appear in the Vulnerabilities Workbench. If this param is omitted it defaults to not include the imported results.

File: src/main/java/com/tenable/io/api/users/models/User.java [hunk 2]
```diff
@@ -20,7 +20,7 @@
     private int loginFailCount;
     private int lastLoginAttempt;
     private UUID uuid;
-    private int aggregate;
+    private boolean aggregate;
```
Comment (tenableAndrew) [inline src/main/java/com/tenable/io/api/users/models/User.java]: The service response changed from returning an into to returning a bool, which was causing errors in user tests.

File: src/main/java/com/tenable/io/api/scans/ScansApi.java (hunk 1 above)
Comment (pierresc) [reply src/main/java/com/tenable/io/api/scans/ScansApi.java]: Should we provide an overload without the aggregate param that defaults it to 1?

Comment (tenableAndrew) [reply src/main/java/com/tenable/io/api/scans/ScansApi.java]: That would probably be safest. UI behavior defaults to setting this to 1.

**Pull request merged**